src/aics-image-viewer/shared/utils/sliderValuesToImageValues.ts

```diff
@@ -32,8 +32,8 @@ export const gammaSliderToImageValues = (sliderValues: [number, number, number])
 // Density and brightness are overloaded for the two supported rendering modes in the volume viewer.
 // These formulae are somewhat ad-hoc based on what will subjectively look reasonable in the viewer, and should be considered tweakable.
 // vole-core expects to see numbers from 0..1 for density, brightness, and alpha.
-export const densitySliderToImageValue = (sliderValues: number, _isPT: boolean): number => +sliderValues / 100.0;
+export const densitySliderToImageValue = (sliderValues: number): number => +sliderValues / 100.0;
```

Comment:
Is the `+{number}` coercion necessary? Are there ever instances where `sliderValues` is not a number?